homeassistant/components/amberelectric/strings.json

```diff
@@ -1,25 +1,61 @@
 {
   "config": {
+    "error": {
+      "invalid_api_token": "Invalid API key",
+      "no_site": "No site provided",
+      "unknown_error": "Unexpected error"
+    },
     "step": {
-      "user": {
-        "data": {
-          "api_token": "[%key:common::config_flow::data::api_token%]",
-          "site_id": "Site ID"
-        },
-        "description": "Go to {api_url} to generate an API key"
-      },
       "site": {
         "data": {
           "site_id": "Site NMI",
           "site_name": "Site Name"
         },
         "description": "Select the NMI of the site you would like to add"
+      },
+      "user": {
+        "data": {
+          "api_token": "API token",
+          "site_id": "Site ID"
+        },
+        "description": "Go to {api_url} to generate an API key"
+      }
+    }
```

Comment:
who do we change the order of the config flow strings?

Reply:
Auto formatter probably - it's looking pretty alphabetical now.

Reply:
It sure looks alphabetical